string-pokemonize.lisp

```diff
@@ -20,7 +20,7 @@
                        (DeBuG 1)
                        (sAfEtY 1)))
     (FlEt ((pOkEmOnIzE (cHaR)
-             (dEcLaRe (ExTeNdEd-cHaR ChAr))
+             (dEcLaRe (ChArAcTeR ChAr))
```

Comment:
`(dEcLaRe (ChArAcTeR ChAr))`
should be
`(dEcLaRe (ChArAcTeR cHaR))`
which in turn forces a case change for all characters after this line.

In order to merge this, I need the case of the code to be consistent. Please format the file using the pokemonize case and reupload it.